agenta-cli/agenta/cli/main.py

```diff
@@ -171,18 +172,17 @@ def init(app_name: str, backend_host: str):
 
         filtered_org = None
         if where_question == "On agenta cloud":
-            which_organization = questionary.select(
-                "Which organization do you want to create the app for?",
-                choices=[
-                    f"{org.name}: {org.description}" for org in user_organizations
-                ],
-            ).ask()
+            if not organisation:
+                which_organization = questionary.select(
+                    "Which organization do you want to create the app for?",
+                    choices=[
+                        f"{org.name}: {org.description}" for org in user_organizations
+                    ],
+                ).ask()
+                organisation = which_organization.split(":")[0]
+
             filtered_org = next(
```

Comment:
bad naming here. I would call `organization` above `organization_name` and `filtered_org` `organization`.
Also, since that the user is giving the organization name raw, we need to add some exception handling here. What would happen if the organization that he has provided does not exist.